From c941d74b3745550fc886ebf20e40725aa4722f5c Mon Sep 17 00:00:00 2001
From: Oz N Tiram <oz.tiram@gmail.com>
Date: Tue, 20 Aug 2019 22:25:57 +0200
Subject: [PATCH 05/13] Remove all shebangs from Python modules and checker

As discussed, Python modules which aren't intended to be invoked
as scripts should not include a shebang line.

Update CONTRIBUTING.md and the checker script.
This script now includes a list SKIP_FILES for files that
should not be checked for boilerplate template.
The tests will now fail if a Python module has a shebang line.
Scripts which should have a shebang line and exists in the directory
`hack` can be ignored by adding them to the SKIP_FILES list.
---
 watch/watch.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/watch/watch.py b/watch/watch.py
index 5966eac..77b9794 100644
--- a/watch/watch.py
+++ b/watch/watch.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # Copyright 2016 The Kubernetes Authors.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
-- 
2.25.1

